var/da/da_obs_io/da_read_omb_tmp.inc

```diff
@@ -361,10 +361,8 @@ subroutine da_read_omb_tmp(filename,unit_in,num,obs_type_in,nc,if_wind_sd)
          if (num_obs > 0) then
             do n = 1, num_obs    
                read(unit_in,'(2i8)') levels, ifgat
-               if (if_write) then
-                  write(omb_unit,'(2i8)')levels, ifgat
-                  num = num + 1
-               end if
+               if (if_write) write(omb_unit,'(2i8)')levels, ifgat
+               num = num + 1
```

Comment:
Looks like all obs types have 'num=num+1' embedded within if block. What is the reason to move it outside if block for these 3 places?

Reply:
Jake, you are right. What I meant to update was not da_read_omb_tmp.inc, but da_read_obs_chem_sfc.inc. 
Now I reverted da_read_omb_tmp.inc and pushed da_read_obs_chem_sfc.inc, instead. Please review it. Thanks!